docs/conf.py

```diff
@@ -52,6 +52,8 @@
 autoconf_min_version = f"{ompi_data['autoconf_min_version']}"
 libtool_min_version = f"{ompi_data['libtool_min_version']}"
 flex_min_version = f"{ompi_data['flex_min_version']}"
+mpi_standard_version = f"{ompi_data['mpi_standard_version']}"
+mpi_standard_subversion = f"{ompi_data['mpi_standard_subversion']}"
```

Comment:
I think you missed a change at the bottom of this file to add the new vars to the prolog.

Reply:
Yup I missed that, thanks